cmd/gossamer/main.go

```diff
@@ -157,11 +112,10 @@ func initNode(ctx *cli.Context) error {
 
 	err = loadGenesis(ctx)
 	if err != nil {
-		log.Error("error loading genesis state", "error", err)
+		log.Error("[gossamer] error loading genesis state", "error", err)
 		return err
 	}
 
-	log.Info("🕸\t Finished initializing node!")
```

Comment:
it would be nice to leave this

Reply:
It provides no new information and the last action before is "Genesis data loaded", which is the only action during initialization. It made more sense the "initializing the node" would actually be happening in `makeNode`, which is where we are initializing all of the node services, here we are just loading genesis data.

Reply:
okay, well the command is `init`, so I figured it would make sense to let the user know when that's done, but it's fine